reconcile/utils/mr/auto_promoter.py

```diff
@@ -34,6 +44,47 @@ def title(self):
         return (f'[{self.name}] openshift-saas-deploy automated '
                 f'promotion {digest}')
 
+    @staticmethod
+    def init_promotion_data(channel: str,
+                            promotion: Mapping[str, Any]) -> Dict[str, Any]:
+        psc = ParentSaasConfigPromotion(
+            parent_saas=promotion["saas_file"],
+            target_config_hash=promotion[TARGET_CONFIG_HASH]
+        )
+        return {
+            "channel": channel,
+            "data": [
+                asdict(psc)
+            ]
+        }
+
+    @staticmethod
+    def process_promotion(promotion,
+                          target_promotion,
+                          target_channels):
+
+        # Existent subscribe data channel data
+        promotion_data = {v["channel"]: v["data"]
+                          for v in target_promotion.get('promotion_data', [])
+                          if v["channel"] in target_channels}
+
+        for channel in target_channels:
+            modified = False
+            channel_data = promotion_data.get(channel)
+            if channel_data is None:
+                channel_data = \
+                    AutoPromoter.init_promotion_data(channel, promotion)
+                modified = True
+            else:
+                for item in channel_data:
+                    if item["type"] == ParentSaasConfigPromotion.TYPE:
+                        psc = ParentSaasConfigPromotion(**item)
+                        if psc.target_config_hash != \
+                                promotion[TARGET_CONFIG_HASH]:
+                            modified = True
```

Comment:
shouldn't there be some target_config modification going on here?

Reply:
same as the other comment, don't know what I was thinking when I did this ...